model/src/main/java/com/ge/predix/acs/rest/AttributeConnector.java

```diff
@@ -8,14 +8,19 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel(description = "Connector configuration for external resource or subject atributes.")
 public class AttributeConnector {
-    private boolean isActive;
+    private boolean isActive = false;
 
     @JsonProperty(required = false)
     private int maxCachedIntervalMinutes = 480; //default value
 
     private Set<AttributeAdapterConnection> adapters;
 
+    @ApiModelProperty(value = "A flag to enable or disable attribute connector. Disabled by default")
```

Comment:
Might be more clearly worded as "A flag to enable or disable the retrieval of remote attributes. Disabled by default."?